src/test/data/GemCommandTest/sem1GEHmodules.json

```diff
@@ -0,0 +1,83 @@
+{
+  "modules" : [ {
+    "tagged" : [ "GEH" ],
```

Comment:
I think you can just use the data in found in main/resources/data/GEM, since all the modules are the same

Reply:
wah i honestly didnt realise this HAHA ok i'll use those instead so no repeating json files